docs/howto/getting-started/change-cust-billing.md

```diff
@@ -0,0 +1,31 @@
+# How to change company information in your account
+
+Via {{gui}} <https://{{gui_domain}}> you can change contact person, adress, company name, purchase order or add c/o adress. This guide will show you how.
+
+## Prerequisites
+
+You need a Cleura account.
+
+[Guide how to create a new account](https://docs.cleura.cloud/howto/getting-started/create-account)
+
+## How to change in Cleura Controlpanel
+
+To get started, navigate to <https://{{gui_domain}}>.
+Log in and click on the _Profile_ button at the top right like picture below.
+
+![picture-profile](assets/change-cust-billing/picture-profile.png)
+
+You will now see Account settings page.
+Choose _Customer Info_ tab in which you can change and manage your Customer information.
+
+![picture-customerinfo](assets/change-cust-billing/picture-customerinfo2.png)
+
+Finished your changes by clicking the green _Update_ button in the bottom
+
+![picture-changecustomerinfo](assets/change-cust-billing/picture-changecustomerinfo2.png)
+
+## **How to change organization number**
+
+This can not be done in {{gui}}
+
+This requires that you send in a [transfer form](https://kb.citynetwork.eu/kb/how-to-articles/general/transfer-account-ownership-to-another-person-or-organisation) to support@citycloud.com.
```

Comment:
We don't backlink from the new docs to the old knowledge base. Add the transfer form as an asset like you did with the screenshots, and link to that.

Also, use:

```markdown
to [support@{{brand_domain}}](mailto:support@{{brand_domain}}).
```